Make run command is broken

After setting everything up (tf IAC, python envs, env vars), executing `make run pipeline=training` fails with this error message:
```console
luegnix-gcp on  main via 🐍 v3.7.12 on ☁️  (eu-central-1) on ☁️  asterix.versus.caesar@proton.me 
❯ make run pipeline=training 
/Users/hans.rauer/Repos/luegnix-gcp/.venv/lib/python3.7/site-packages/kfp/v2/compiler/compiler.py:1266: FutureWarning: APIs imported from the v1 namespace (e.g. kfp.dsl, kfp.components, etc) will not be supported by the v2 compiler since v2.0.0
  category=FutureWarning,

WARNING: gsutil rsync uses hashes when modification time is not available at
both the source and destination. Your crcmod installation isn't using the
module's C extension, so checksumming will run very slowly. If this is your
first rsync since updating gsutil, this rsync can take significantly longer than
usual. For help installing the extension, please see "gsutil help crcmod".

Building synchronization state...
If you experience problems with multiprocessing on MacOS, they might be related to https://bugs.python.org/issue33725. You can disable multiprocessing by editing your .boto config or by adding the following flag to your command: `-o "GSUtil:parallel_process_count=1"`. Note that multithreading is still available even if you disable multiprocessing.

Starting synchronization...
If you experience problems with multiprocessing on MacOS, they might be related to https://bugs.python.org/issue33725. You can disable multiprocessing by editing your .boto config or by adding the following flag to your command: `-o "GSUtil:parallel_process_count=1"`. Note that multithreading is still available even if you disable multiprocessing.

usage: main.py [-h] [--template_path TEMPLATE_PATH]
               [--enable_caching ENABLE_CACHING]
main.py: error: unrecognized arguments: --pipeline=./training.json
make: *** [run] Error 2
```
It the included targets `make compile` and `make sync-assets` work as expected. It clearly fails, because the python `trigger` code does not define the option `--pipeline`. Looking at the code, this is not unexpected:
```python
def sandbox_run(args: List[str] = None) -> aiplatform.PipelineJob:
    """Trigger a Vertex Pipeline run from a (local) compiled pipeline definition.
    Returns the PipelineJob object of the triggered pipeline run.
    Usage: python main.py --template_path=pipeline.json --enable_caching=true
    """
    logging.basicConfig(level=logging.DEBUG)

    parser = argparse.ArgumentParser()
    parser.add_argument(
        "--template_path", help="Path to the payload JSON file", type=str
    )
    parser.add_argument("--enable_caching", type=str, default=None)

    # Get commandline args
    args = parser.parse_args(args)

    # If empty value for enable_caching provided on commandline default to None
    if args.enable_caching == "":
        args.enable_caching = None

    payload = {
        "attributes": {
            "template_path": args.template_path,
            "enable_caching": args.enable_caching,
        }
        # "data" omitted as pipeline params are taken from the default args
        # in compiled JSON pipeline
    }

    return trigger_pipeline_from_payload(payload)


if __name__ == "__main__":
    sandbox_run()

```

That's exactly what I did right now, I also drew that connection :D : 

```console
 luegnix-gcp on  main via 🐍 v3.7.12 on ☁️  (eu-central-1) on ☁️  asterix.versus.caesar@proton.me took 7s 
❯ make run pipeline=training 
/Users/hans.rauer/Repos/luegnix-gcp/.venv/lib/python3.7/site-packages/kfp/v2/compiler/compiler.py:1266: FutureWarning: APIs imported from the v1 namespace (e.g. kfp.dsl, kfp.components, etc) will not be supported by the v2 compiler since v2.0.0
  category=FutureWarning,

WARNING: gsutil rsync uses hashes when modification time is not available at
both the source and destination. Your crcmod installation isn't using the
module's C extension, so checksumming will run very slowly. If this is your
first rsync since updating gsutil, this rsync can take significantly longer than
usual. For help installing the extension, please see "gsutil help crcmod".

Building synchronization state...
If you experience problems with multiprocessing on MacOS, they might be related to https://bugs.python.org/issue33725. You can disable multiprocessing by editing your .boto config or by adding the following flag to your command: `-o "GSUtil:parallel_process_count=1"`. Note that multithreading is still available even if you disable multiprocessing.

Starting synchronization...
If you experience problems with multiprocessing on MacOS, they might be related to https://bugs.python.org/issue33725. You can disable multiprocessing by editing your .boto config or by adding the following flag to your command: `-o "GSUtil:parallel_process_count=1"`. Note that multithreading is still available even if you disable multiprocessing.

INFO:google.cloud.aiplatform.pipeline_jobs:Creating PipelineJob
Traceback (most recent call last):
  File "/Users/hans.rauer/Repos/luegnix-gcp/.venv/lib/python3.7/site-packages/google/api_core/grpc_helpers.py", line 67, in error_remapped_callable
    return callable_(*args, **kwargs)
  File "/Users/hans.rauer/Repos/luegnix-gcp/.venv/lib/python3.7/site-packages/grpc/_channel.py", line 946, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/Users/hans.rauer/Repos/luegnix-gcp/.venv/lib/python3.7/site-packages/grpc/_channel.py", line 849, in _end_unary_response_blocking
    raise _InactiveRpcError(state)
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
        status = StatusCode.PERMISSION_DENIED
        details = "Vertex AI API has not been used in project 582867146070 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/aiplatform.googleapis.com/overview?project=582867146070 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry."
        debug_error_string = "{"created":"@1674122049.786475000","description":"Error received from peer ipv6:[2a00:1450:4001:812::200a]:443","file":"src/core/lib/surface/call.cc","file_line":904,"grpc_message":"Vertex AI API has not been used in project 582867146070 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/aiplatform.googleapis.com/overview?project=582867146070 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.","grpc_status":7}"
>

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/hans.rauer/.pyenv/versions/3.7.15/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/Users/hans.rauer/.pyenv/versions/3.7.15/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/hans.rauer/Repos/luegnix-gcp/pipelines/trigger/main.py", line 222, in <module>
    sandbox_run()
  File "/Users/hans.rauer/Repos/luegnix-gcp/pipelines/trigger/main.py", line 218, in sandbox_run
    return trigger_pipeline_from_payload(payload)
  File "/Users/hans.rauer/Repos/luegnix-gcp/pipelines/trigger/main.py", line 77, in trigger_pipeline_from_payload
    enable_caching=payload["attributes"]["enable_caching"],
  File "/Users/hans.rauer/Repos/luegnix-gcp/pipelines/trigger/main.py", line 132, in trigger_pipeline
    network=network,
  File "/Users/hans.rauer/Repos/luegnix-gcp/.venv/lib/python3.7/site-packages/google/cloud/aiplatform/pipeline_jobs.py", line 281, in submit
    pipeline_job_id=self.job_id,
  File "/Users/hans.rauer/Repos/luegnix-gcp/.venv/lib/python3.7/site-packages/google/cloud/aiplatform_v1/services/pipeline_service/client.py", line 1068, in create_pipeline_job
    response = rpc(request, retry=retry, timeout=timeout, metadata=metadata,)
  File "/Users/hans.rauer/Repos/luegnix-gcp/.venv/lib/python3.7/site-packages/google/api_core/gapic_v1/method.py", line 145, in __call__
    return wrapped_func(*args, **kwargs)
  File "/Users/hans.rauer/Repos/luegnix-gcp/.venv/lib/python3.7/site-packages/google/api_core/grpc_helpers.py", line 69, in error_remapped_callable
    six.raise_from(exceptions.from_grpc_error(exc), exc)
  File "<string>", line 3, in raise_from
google.api_core.exceptions.PermissionDenied: 403 Vertex AI API has not been used in project 582867146070 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/aiplatform.googleapis.com/overview?project=582867146070 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.
make: *** [run] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -36,7 +36,7 @@ sync-assets: ## Sync assets folder to GCS. Must specify pipeline=<training|predi
 run: ## Compile pipeline, copy assets to GCS, and run pipeline in sandbox environment. Must specify pipeline=<training|prediction>. Optionally specify enable_pipeline_caching=<true|false> (defaults to default Vertex caching behaviour)
 	@ $(MAKE) compile && \
 	$(MAKE) sync-assets && \
-	pipenv run python -m pipelines.trigger.main --pipeline=./$(pipeline).json --enable_caching=$(enable_pipeline_caching)
+	pipenv run python -m pipelines.trigger.main --template_path=./$(pipeline).json --enable_caching=$(enable_pipeline_caching)
 
 e2e-tests: ## Compile pipeline, copy assets to GCS, and perform end-to-end (E2E) pipeline tests. Must specify pipeline=<training|prediction>. Optionally specify enable_pipeline_caching=<true|false> (defaults to default Vertex caching behaviour)
 	@ $(MAKE) compile && \

--- a/pipelines/trigger/main.py
+++ b/pipelines/trigger/main.py
@@ -195,7 +195,7 @@ def sandbox_run(args: List[str] = None) -> aiplatform.PipelineJob:
 
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        "--template_path", help="Path to the payload JSON file", type=str
+        "--template_path", help="Path to the compiled pipeline (JSON)", type=str
     )
     parser.add_argument("--enable_caching", type=str, default=None)
 

